www/class/centreonContactgroup.class.php

```diff
@@ -328,22 +332,30 @@ public function syncWithLdap()
                     $this->insertLdapGroup('[' . $ldapRow['ar_id'] . ']' . $toInsertGroup);
                 }
 
-                $res = $this->db->query("SELECT cg_id, cg_name, cg_ldap_dn FROM contactgroup " .
-                    "WHERE cg_type = 'ldap' AND ar_id = " . $ldapRow['ar_id']);
+                $res = $this->db->prepare(
+                    "SELECT cg_id, cg_name, cg_ldap_dn FROM contactgroup " .
+                    "WHERE cg_type = 'ldap' AND ar_id = :arId"
+                );
+                $res->bindValue(':arId', $ldapRow['ar_id'], \PDO::PARAM_INT);
+                $res->execute();
 
                 while ($row = $res->fetch()) {
```

Comment:
Don't forget to create a db transaction when there is more than one update request. (=> while)